receiver/memcachedreceiver/metadata.yaml

```
@@ -1,41 +1,113 @@
 name: memcachedreceiver
 
 attributes:
+  command:
+    description: The type of command.
+    enum:
+    - get
+    - set
+    - flush
+    - touch
+  direction: 
+    description: Direction of data flow.
+    enum:
+    - sent
+    - received
+  type: 
+    description: hit/miss
+    enum:
+    - hit
+    - miss
+  operation:
+    description: The type of operation.
+    enum:
+    - increment
+    - decrement
+    - get
+  usage_type:
```

```suggestion
  state:
```

See conventions for "usage" metric: https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-naming

---

Updated